src/conf_mode/protocols_mpls.py

```diff
@@ -121,19 +121,31 @@ def apply(mpls):
         system_interfaces = []
         system_interfaces.append(((os.popen('sysctl net.mpls.conf').read()).split('\n')))
         del system_interfaces[0][-1]
-        for configured_interface in mpls['interface']:
-            for system_interface in system_interfaces[0]:
-                if configured_interface in system_interface:
-                    call(f'sysctl -wq net.mpls.conf.{configured_interface}.input=1')
-                elif system_interface.endswith(' = 1'):
-                    system_interface = system_interface.replace(' = 1', '=0')
-                    call(f'sysctl -wq {system_interface}')
+        system_interfaces = system_interfaces[0]
```

Comment:
Why not merge line 124 and 125?

```
for system_interface in system_interfaces[0]:
```

Reply:
I can do so, I just don't like referencing the element. If you'd prefer "for system_interface in system_interfaces[0]:" then I absolutely will defer to that.

Reply:
Well "under the hood" it does the same - it only spares one line of code which makes it simpler to read.

Reply:
Yeah, and that's fine. It does save a line. I just didn't know which was preferred. Either works for me. Thank you :)